### PR TITLE
Add spikeglx lfp interface

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
@bendichter Added LFP interface for SpikeGLX format.

Also fixed a bug in `NwbConverter.run_conversion` where it always makes a new NWBFile object, even if you were trying to pass one in to be operated on and returned.

### CODE REVIEW

File: nwb_conversion_tools/datainterfaces/spikeglxdatainterface.py [hunk 1]
```diff
@@ -56,4 +56,23 @@ def get_metadata(self):
...
+
+    def get_metadata(self):
+        metadata = dict(
+            Ecephys=dict(
+                LFPElectricalSeries=dict(
+                    name="LFP",
+                    description="Local field potential signal."
+                )
+            )
+        )
```
Comment (bendichter) [inline nwb_conversion_tools/datainterfaces/spikeglxdatainterface.py:76]: This seems generic. Can it be put [here](https://github.com/catalystneuro/nwb-conversion-tools/blob/452d456d15f0d2107077f28ad73c7b62512af454/nwb_conversion_tools/baselfpextractorinterface.py#L15) instead?

Comment (CodyCBakerPhD) [reply nwb_conversion_tools/datainterfaces/spikeglxdatainterface.py:76]: Yeah, that's the better place for it. Updated!

File: nwb_conversion_tools/datainterfaces/spikeglxdatainterface.py [hunk 2]
```diff
@@ -56,4 +56,11 @@ def get_metadata(self):
                 )
             )
         )
+
         return ecephys_metadata
+
+
+class SpikeGLXLFPInterface(BaseLFPExtractorInterface):
+    """Primary data interface class for converting the high-pass (ap) SpikeGLX format."""
```
Comment (bendichter) [inline nwb_conversion_tools/datainterfaces/spikeglxdatainterface.py:64]: usually it's the low pass that's called "lfp"

Comment (CodyCBakerPhD) [reply nwb_conversion_tools/datainterfaces/spikeglxdatainterface.py]: Whoops, that would left over from copying/pasting from the RecordingInterface. Fixed!

**Review by bendichter — APPROVED** [review]

**Pull request merged**